openmdao/components/exec_comp.py

```diff
@@ -74,28 +75,10 @@ class ExecComp(ExplicitComponent):
         Default is None, which means units are provided for variables individually.
     complex_stepsize : double
         Step size used for complex step which is used for derivatives.
+    _manual_decl_partials : bool
+        If True, at least one partial has been declared by the user.
```

Comment:
I think `requires_fd` is missing from here.

Reply:
fixed